AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.m

```diff
@@ -207,7 +207,14 @@ - (void)removeAuthToken:(nullable NSString *)authToken {
   if (history) {
     MSLogDebug([MSAppCenter logTag], @"Retrieved history state.");
   } else {
-    MSLogWarning([MSAppCenter logTag], @"Failed to retrieve history state or none was found.");
+    if (decryptedData) {
+      MSLogWarning([MSAppCenter logTag], @"Failed to unarchive decryptedData.");
+    } else if (encryptedData) {
+      MSLogWarning([MSAppCenter logTag], @"Failed to get decryptedData.");
+    } else {
+      MSLogInfo([MSAppCenter logTag], @"No token history was found.");
```

Comment:
Line 202 should also need this one.

Reply:
Line 202 is the positive path. is that necessary?